engines/sci/graphics/text32.cpp

```diff
@@ -336,6 +341,9 @@ void GfxText32::drawTextBox() {
 	charIndex = 0;
 	uint nextCharIndex = 0;
 	while (*text != '\0') {
+		// We need to check for Korean-EUCKR every line
```

Comment:
Why do you need to check for Korean in each line? Could you store the result of this function in a variable and check that instead?